lib/reverse_sentence.rb

```diff
@@ -1,6 +1,41 @@
-# A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
-def reverse_sentence(my_sentence)
-  raise NotImplementedError
-end
+# Time complexity: 0(n) - no nested loops; each single loop runs the length of the string
+# Space complexity: 0(1) - characters are reversed in place
+def reverse_sentence(my_string)
```

Comment:
👍 Outstanding implementation.  I do suggest below that you create a helper method.